content/Tjenester/fiksprotokoll/_index.md

```diff
@@ -5,8 +5,9 @@ aliases: [/fiks-platform/tjenester/fiksprotokoll, /fiks-plattform/tjenester/fiks
 ---
 
 ## Kort beskrivelse
-Fiks protokoll er en tjenestegruppe som inneholder flere tjenester. Felles for disse tjenestene er at de gjelder meldinger som sendes over [Fiks IO](https://ks-no.github.io/tjenester/fiksprotokoll/fiksio), for sikker maskin-til-maskin integrasjon. 
-Tjenestene består både av et sett av protokoller med meldingstyper, f.eks. Fiks Arkiv og Fiks Plan, og et miljø for å administrere systemer som implementerer disse protokollene over Fiks IO.
+
+Fiks protokoll er en tjenestegruppe som inneholder flere tjenester, hvor blant annet [Fiks IO](https://ks-no.github.io/tjenester/fiksprotokoll/fiksio) er tjenesten som sørger for den asynkrone maskin-til-maskin meldingsutvekslingen.
+I tillegg til meldingsutveksling via Fiks IO sørger Fiks Protokoll for validering av meldingstyper innenfor protokollen (f.eks. Fiks Arkiv eller Fiks Plan), godkjenning av avsender-system og mottaker-system osv.  
```

Comment:
Alternativt forslag.

I tillegg til meldingsutveksling via Fiks IO validerer Fiks Protokoll at det kun er gyldige meldingstyper som sendes for de ulike protokollene (f.eks Fiks Arkiv eller Fiks Plan). Videre at det kun sendes meldinger mellom avsender- og mottakersystem som er forhåndsgodkjent av systemadministratorene.